armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -257,6 +255,11 @@ After=network.target startup-checks.service
 Requires=startup-checks.service
 [Service]
 ExecStart=/usr/bin/bitcoind -daemon -conf=/etc/bitcoin/bitcoin.conf 
+# Note: we need this ExecStartPost command to prepare an .env file
+# in expected format for electrs to have the RPCPASSWORD in its
+# environment. If we contributed a --cookie-file flag to upstream
+# electrs that could parse the .cookie file, we would not need this.
+ExecStartPost=/bin/bash -c "sleep 5 && echo \"RPCPASSWORD=$(tail -c +12 /mnt/ssd/bitcoin/.bitcoin/testnet3/.cookie)\" > /mnt/ssd/bitcoin/.bitcoin/testnet3/.cookie.env"
```

Comment:
Loading the content of the bitcoind cookie file from standard location would mean that all locations have to be adjusted when switching from testnet to mainnet. This means that the script `base/scripts/set-bitcoin-network.sh` needs to be extended as well.

Instead, I propose to store the `.cookie` always in the same location, regardless of the network that currently is active. For example with this configuration in `/etc/bitcoin/bitcoin.conf`:
```
rpccookiefile=/mnt/ssd/bitcoin/.bitcoin/.cookie
```

Reply:
Great idea, done.